sensor_test.py

```diff
@@ -55,6 +56,38 @@ def test_set_sensor_name(get_sensor_info, set_sensor_name):
     ), f"Sensor name was not updated, expected name:{new_sensor_name}"
 
 
+def test_set_empty_sensor_name(get_sensor_info, set_sensor_name):
+    """
+    1. Get original sensor name.
+    2. Set sensor name to an empty string.
+    3. Validate that sensor responds with an error.
+    4. Get current sensor name.
+    5. Validate that sensor name didn't change.
+    """
+
+    log.info("Get original sensor name")
+    sensor_info = get_sensor_info()
+    original_sensor_name = sensor_info.name
+
+    log.info("Set sensor name to an empty string")
+    set_empty_sensor_name = set_sensor_name("")

```

Comment:
A more appropriate variable name would be something like `set_empty_sensor_name_response`.